dashboard/src/components/AppRoot.js

```diff
@@ -49,6 +51,10 @@ class AppRoot extends React.PureComponent {
               />
               <Route component={NotFoundView} />
             </Switch>
+            <Switch>
+              <UnauthenticatedRoute exact path="/signin" />
+              <AuthInvalidRoute component={AuthInvalidDialog} />
+            </Switch>
```

Comment:
🤔 can be deferred until later but I'm not exactly in love with this. The intent is not immediately clear to me and will break the next time we add another unauthenticated route to the application.

Reply:
This approach appears to be the only option for creating a route with exceptions. (https://stackoverflow.com/questions/45591165/react-route-component-handle-all-routes-except-for-login)

Reply:
Our other option would be to render `<AuthInvalidRoute component={AuthInvalidDialog} />` somewhere on every page in the main switch.